interactive_engine/bin/giectl.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+#
+# interactive_engine command tool
+
+readonly SCRIPT_DIR=$(cd "$(dirname "$0")";pwd)
+readonly WORKSPACE=${SCRIPT_DIR}/../
+
+err() {
+  echo -e "${RED}[$(date +'%Y-%m-%dT%H:%M:%S%z')]: [ERROR]${NC} $*" >&2
+}
+
+warning() {
+  echo -e "${YELLOW}[$(date +'%Y-%m-%dT%H:%M:%S%z')]: [WARNING]${NC} $*" >&1
+}
+
+log() {
+  echo -e "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&1
+}
+
+
+##########################
+# Output useage information.
+# Globals:
+#   None
+# Arguments:
+#   None
+##########################
+usage() {
```

Comment:
need to revise

Reply:
need to revise, could you give the detailed sub-command?

Reply:
fixed